protocols/irods.md

```diff
@@ -22,49 +26,143 @@ Connection profiles can be installed from *Preferences → Profiles*.
+    <string>pam_password</string>
+```
+
+`pam` is an alias for `pam_password`. It is provided for backward compatibility.
+
+#### Configuring the iRODS Zone
+
+The zone of the target server must be specified before connecting to it. To do that, set the **Region** value in the [connection profile](index.md#connection-profiles).
+
+```
+    <key>Region</key>
+    <string>zone_name</string>
+```
 
-#### Authentication With PAM Scheme
+### Secure Communication
 
-To authenticate with PAM, prefix the username with PAM:or set it in the [connection profile](index.md#connection-profiles) using as
+To require secure communication, you must first set the **Client Server Negotiation** value to `CS_NEG_REQUIRE` in the [connection profile](index.md#connection-profiles).
 
 ```
-      <key>Authorization</key>
-      <string>PAM</string>
+    <key>Properties</key>
+    <dict>
+        <key>Client Server Negotiation</key>
+        <string>CS_NEG_REQUIRE</string>
+    </dict>
 ```
 
-### Resource Server
+Set the value to `CS_NEG_DONT_CARE` to let the server decide whether to use secure communication. Set it to `CS_NEG_REFUSE` to require an insecure channel. If the client and server cannot agree on how to proceed, the connection will fail and result in an error.
 
-You can specify a non-default resource server in the Region key of the [connection profile](index.md#connection-profiles) using a colon-separated syntax (Issue [#8721](https://github.com/iterate-ch/cyberduck/issues/8721)) such as
+Next, configure the encryption values to match those of the server. Consult the administrator of the iRODS zone to obtain the correct values.
 
 ```
-   	<key>Region</key>
-   	<string>iplant:storageresourcename</string>
+    <key>Properties</key>
+    <dict>
+        <key>Encryption Algorithm</key>
+        <string>AES-256-CBC</string>
+        
+        <key>Encryption Key Size</key>
+        <string>32</string>
+        
+        <key>Encryption Salt Size</key>
+        <string>8</string>
+        
+        <key>Encryption Hash Rounds</key>
+        <string>16</string>
+    </dict>
+```
+
+#### Storage Resource
+
+To upload files to a specific resource, set the **Destination Resource** value in the [connection profile](index.md#connection-profiles). The specified resource must be the root of a resource hierarchy. This property supersedes any resource defined by the **Region** property.
+
+```
+    <key>Properties</key>
+    <dict>
+        <key>Destination Resource</key>
+        <string>resource_name</string>
+    </dict>
+```
+
+For backward compatibility, it can also be set by appending the name of the resource to the **Region**'s value.
+
+```
+    <key>Region</key>
+    <string>zone_name:resource_name</string>
 ```
 
 ## Transfers
 
-### Use Parallel Connections
+### Parallel Transfer
+
+Large file uploads will use parallel transfer when the requirements are satisfied. By default, a file must be larger than 32MB in order to trigger parallel transfer.
```

Comment:
up and down?

```suggestion
Large file uploads and downloads will use parallel transfer when the requirements are satisfied. By default, a file must be larger than 32MB in order to trigger parallel transfer.
```

Reply:
ah, very good.

Reply:
I don't know if cyberduck uses an internal threshold for parallel reads though.